scripts/list-releases.sh

```diff
@@ -0,0 +1,41 @@
+ #!/bin/bash
+
+if [ "$1" = "" ]; then
+    echo "Error: please provide provider parameter"
+    echo "./list-releases {provider} {release_active_tag(optional)}"
+    exit 1
+fi
+
+opsctl list installations > list-of-installations.txt
+gsctl list endpoints > list-of-endpoints.txt
+
+provider=$1
+active_release=$2
+while read installation; do
+  #echo $installation
+  #inallation_data=$(echo $installation | tr -d " ")
+  IFS=' ' read -ra inallation_data <<< "$installation"
+  if [ "${inallation_data[1]}" == "$provider" ]
+  then
+		echo "-----${inallation_data[0]}-----"
+		gsctl select endpoint "${inallation_data[0]}"
+		
+		if [ "$active_release" = "" ]
+		then
+			gsctl list releases
+		else
+			gsctl list releases > list-of-releases.txt
+			while read releases; do
+				IFS=' ' read -ra release_data <<< "$releases"
+				if [ "${release_data[0]}" == "$active_release" ]
+			    then
+			    	echo "Release $active_release is ${release_data[1]}"
+			    fi
+		done <list-of-releases.txt
```

Comment:
I feel like this could be possibly simplified quite a bit, but what does it print out? The active releases per installation?

Reply:
yeah it prints active releases per installation, or the one u r looking for